.circleci/config.yml

```diff
@@ -33,19 +87,40 @@ jobs:
             - deps
             - ~/.mix
 
-      #TODO: uncomment after fixing dialyzer issues
-      # - restore_cache:
-      #     keys:
-      #       - v1-plt-cache-{{ arch }}-{{ checksum "mix.lock" }}
-      #       - v1-plt-cache-{{ arch }}
-      #       - v1-plt-cache
+  dialyzer:
+    working_directory: ~/ethereum
+    docker:
+      - image: elixir:latest
+    steps:
+      - run: apt-get update; apt-get -y install libtool autoconf libgmp3-dev
+      - checkout
+      - run: git submodule sync --recursive
+      - run: git submodule update --recursive --init
+
+      - restore_cache:
+         keys:
+           - v1-dependency-cache-{{ arch }}-{{ checksum "mix.lock" }}
+           - v1-dependency-cache-{{ arch }}
+           - v1-dependency-cache
 
-      # - run: mix dialyzer --plt
+      - run: mix local.rebar --force
+      - run: mix local.hex --force
+      - run: mix deps.get
+
+      - run: mix dialyzer --plt
```

Comment:
have you missed mix dialyzer --halt-exit-status?

Reply:
Right. Add this line and we're good.